src/file-generator/generator/file.go

```
@@ -0,0 +1,78 @@
+package generator
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+type PathGenerator interface {
+	Generate(jobName string) string
+}
+
+type JobFilePathGenerator struct{}
+
+func (j JobFilePathGenerator) Generate(jobName string) string {
+	return fmt.Sprintf("/var/vcap/jobs/%s/config", jobName)
+}
+
+// FileGenerator ...
+type FileGenerator struct {
+	FileName          string
+	ComponentName     string
+	FilePathGenerator PathGenerator
+}
+
+func NewFileGenerator(filePath, componentName string, generator PathGenerator) FileGenerator {
+	return FileGenerator{
+		FileName:          filePath,
+		ComponentName:     componentName,
+		FilePathGenerator: generator,
+	}
+}
+
+func (f FileGenerator) Parse() (map[string]string, error) {
+	jsonFile, err := os.Open(f.FileName)
+	if err != nil {
+		return nil, err
+	}
+	defer jsonFile.Close()
+
+	byteValue, _ := ioutil.ReadAll(jsonFile)
```

I imagine you won't swallow this error in the final version of the PR